pkg/observability/config.go

```diff
@@ -60,4 +62,8 @@ type StackdriverConfig struct {
 	// Allows for providing a real Google Cloud location when running locally for development.
 	// This is ignored if a real location was found during discovery.
 	LocationOverride string `env:"DEV_STACKDRIVER_LOCATION"`
+
+	ReportingInterval    time.Duration `env:"STACKDRIVER_REPORTING_INTERVAL, default=2m"`
```

Comment:
It'd be good to document these and any constraints they might have relative to eachother.